pkg/sqlite/load_test.go

```diff
@@ -424,6 +424,42 @@ func TestDeprecationAwareLoader(t *testing.T) {
 				},
 			},
 		},
+		{
+			description: "DeprecateTruncateRemoveDeprecatedChannelHeadOnPackageRemoval",
+			fields: fields{
+				bundles: []*registry.Bundle{
+					withBundleImage("quay.io/my/bundle-a", newBundle(t, "csv-a", "pkg-0", []string{"a"}, newUnstructuredCSV(t, "csv-a", ""))),
+					withBundleImage("quay.io/my/bundle-aa", newBundle(t, "csv-aa", "pkg-0", []string{"a"}, newUnstructuredCSV(t, "csv-aa", "csv-a"))),
+					withBundleImage("quay.io/my/bundle-aaa", newBundle(t, "csv-b", "pkg-0", []string{"b"}, newUnstructuredCSV(t, "csv-b", "csv-a"))),
```

Comment:
Is there a reason the image ref is `aaa` but the name is `b`?
```suggestion
					withBundleImage("quay.io/my/bundle-aaa", newBundle(t, "csv-aaa", "pkg-0", []string{"b"}, newUnstructuredCSV(t, "csv-aaa", "csv-a"))),
```

Reply:
Typo, good catch!